src/utils/FetchData.jsx

```diff
@@ -21,3 +21,13 @@ export const fetchComments = (article_id) => {
     return response.data.comments;
   });
 };
+
+export const patchArticleVote = (article_id, votes) => {
+  console.log(article_id, votes);
```

Comment:
Don't forget to remove console.logs once they aren't needed any more - helps to avoid clogging up the console